app/controllers/restaurants_controller.rb

```diff
@@ -0,0 +1,29 @@
+class RestaurantsController < ApplicationController
+
+  def index
+    render json: Restaurant.all
+  end
+
+  def show
+    render json: Restaurant.find(params.require(:id))
+  end
+
+  def create
+    restaurant = Restaurant.create!(name: params.require(:name),
+                          address: params.require(:address),
+                          does_accept_10bis: params.require(:does_accept_10bis),
+                          delivery_sla_in_minutes: params.require(:delivery_sla_in_minutes),
+                          cuisine_id: params.require(:cuisine_id))
+    restaurant.restaurant_reviews_metadatum = RestaurantReviewsMetadatum.create(reviewsCount: 0, avarageScore: 0)
+    render json: restaurant
+  end
+
+  def update
+    render json: Restaurant.update!(id: params.require(:id),
```

Comment:
Usually, when you issue an update request, you only pass the `diff`s, in this case, if I don't pass `param[:name]` I'll be nullifying the field

Reply:
done